[2022-03-15 18:07:45,185] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: gcs_ingestion_dag_v02.download_dataset_task scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-03-15 18:07:45,192] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: gcs_ingestion_dag_v02.download_dataset_task scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-03-15 18:07:45,192] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 18:07:45,192] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2022-03-15 18:07:45,192] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 18:07:45,200] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-01-01 00:00:00+00:00
[2022-03-15 18:07:45,205] {standard_task_runner.py:52} INFO - Started process 778 to run task
[2022-03-15 18:07:45,207] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'gcs_ingestion_dag_v02', 'download_dataset_task', 'scheduled__2020-01-01T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/gcs_ingestion_dag_v02.py', '--cfg-path', '/tmp/tmpgwrmfm6s', '--error-file', '/tmp/tmp8fjxb2sc']
[2022-03-15 18:07:45,207] {standard_task_runner.py:77} INFO - Job 25: Subtask download_dataset_task
[2022-03-15 18:07:45,246] {logging_mixin.py:109} INFO - Running <TaskInstance: gcs_ingestion_dag_v02.download_dataset_task scheduled__2020-01-01T00:00:00+00:00 [running]> on host 3570d6d4f2dc
[2022-03-15 18:07:45,289] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gcs_ingestion_dag_v02
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T00:00:00+00:00
[2022-03-15 18:07:45,290] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-15 18:07:45,291] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://www.dropbox.com/s/a1kl5e35j4o53mz/bandcamp-items-json.zip > /opt/***/bandcamp-items-json.zip']
[2022-03-15 18:07:45,303] {subprocess.py:85} INFO - Output:
[2022-03-15 18:07:55,301] {local_task_job.py:212} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2022-03-15 18:07:55,302] {process_utils.py:124} INFO - Sending Signals.SIGTERM to group 778. PIDs of all processes in the group: [779, 780, 778]
[2022-03-15 18:07:55,302] {process_utils.py:75} INFO - Sending the signal Signals.SIGTERM to group 778
[2022-03-15 18:07:55,302] {taskinstance.py:1408} ERROR - Received SIGTERM. Terminating subprocesses.
[2022-03-15 18:07:55,302] {subprocess.py:99} INFO - Sending SIGTERM signal to process group
[2022-03-15 18:07:55,354] {process_utils.py:70} INFO - Process psutil.Process(pid=779, status='terminated', started='18:07:44') (779) terminated with exit code None
[2022-03-15 18:07:55,354] {process_utils.py:70} INFO - Process psutil.Process(pid=778, status='terminated', exitcode=0, started='18:07:44') (778) terminated with exit code 0
[2022-03-15 18:07:55,355] {process_utils.py:70} INFO - Process psutil.Process(pid=780, status='terminated', started='18:07:44') (780) terminated with exit code None
